cartoframes/auth/credentials.py

```diff
@@ -50,10 +50,11 @@ class Credentials(object):
     """
 
     def __init__(self, creds=None, key=None, username=None, base_url=None,
-                 cred_file=None):
+                 cred_file=None, api_key=None):
         self._key = None
         self._username = None
         self._base_url = None
+        key = key or api_key
```

Comment:
is this because we are deprecating the `key` parameter?

Reply:
Yes. We need to do more work on the Credentials class to update its interface. This only line is to allow the new Credentials interface for now.